### ISSUE TITLE
Sort PDF Templates when generating Letter

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#712]
When generating PDFs from say an account the user is prompted with a list of templates. By default this is unsorted. Sorting these alphabetically would help users find templates.

**Comment by ghermans** [comment]
+1

**Comment by EZS-JD** [comment]
+1

**Comment by amariussi** [comment]
Confirmed in 7.8.2
![templates_not_sorted](https://cloud.githubusercontent.com/assets/8026129/23472094/0bcb3b1e-feac-11e6-85fd-eb156f7f0ffd.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Sort PDF Templates when generating Letter](https://trello.com/c/SeaIpFm4/133-sort-pdf-templates-when-generating-letter)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
